src/firefly/js/api/ApiBuild.js

```diff
@@ -36,6 +36,7 @@ import {ImageMetaDataToolbar} from '../visualize/ui/ImageMetaDataToolbar.jsx';
 import {MultiViewStandardToolbar} from '../visualize/ui/MultiViewStandardToolbar.jsx';
 import {ImageExpandedMode} from '../visualize/iv/ImageExpandedMode.jsx';
 import {ApiExpandedDisplay} from '../visualize/ui/ApiExpandedDisplay.jsx';
+import {ApiFullDisplay} from '../visualize/ui/ApiFullDisplay.jsx';
```

Comment:
This file is missing from the branch, as far as I can tell.

Reply:
just pushed a fix, sorry